src/duties/browserResults.js

```diff
@@ -0,0 +1,57 @@
+let _ = require('lodash/fp')
+
+let hasBrowserErrors = ({ browserResults, fail, message }) => {
+  if (_.isNil(browserResults)) {
+    message('Could not find any browser results')
+    return
+  }
+
+  let { result, summary } = browserResults
+
+  if (result && summary) {
+    let { error, failed, exitCode } = summary
+    if (false === error && 0 === failed && 0 === exitCode) {
+      message('Your PR has no browser errors.  Great job!')
+      return
+    }
+
+    if (failed > 0) {
+      let failedTests = []
+      _.flow(
+        // Get result for each test suite.
+        _.mapValues(r => r),
+        // Flatten test results into one array.
+        _.reduce((flattened, other) => {
+          return flattened.concat(other)
```

Comment:
you can do `(flattened, other) => flattened.concat(other)`, or even better, `(flattened, other) => [...flattened, other]`